deployments/moonriver.diamond.json

```diff
@@ -61,8 +61,11 @@
       "Receiver": "0x5439f8ca43f832DD21a28C5BF038dad4c07ad02c",
       "RelayerCelerIM": "0x6a8b11bF29C0546991DEcD6E0Db8cC7Fda22bA97",
       "ServiceFeeCollector": "0x894b3e1e30Be0727eb138d2cceb0A99d2Fc4C55D",
-      "LiFuelFeeCollector": "0xc4f7A34b8d283f66925eF0f5CCdFC2AF3030DeaE",
-      "TokenWrapper": "0x5215E9fd223BC909083fbdB2860213873046e45d"
+      "LiFuelFeeCollector": "0xc02FFcdD914DbA646704439c6090BAbaD521d04C",
+      "TokenWrapper": "0x5215E9fd223BC909083fbdB2860213873046e45d",
+      "GasRebateDistributor": "",
+      "LiFiDEXAggregator": "",
+      "ReceiverStargateV2": ""
```

Comment:
that is OK, not every contract is deployed on all networks. However, this PR is about deploying the LiFiDEXAggregator to several networks, including moonriver. So it would have been better/correct for you to point out that THAT address is missing in the log file. This is indeed a mistake/error which I just fixed.